.travis.yml

```diff
@@ -26,6 +27,9 @@ matrix:
           dist: trusty
           env: T=normal
         - os: linux
+          compiler: gcc
+          env: T=gssapi
```

Comment:
After the recent changes to this file, maybe just C=--with-gssapi could be added to the linux/normal build?